mlflow/server/js/src/utils/Utils.test.js

```diff
@@ -89,13 +89,13 @@ test("getGitHubRegex", () => {
     ["https://github.com/mlflow/mlflow-apps", ["/github.com/mlflow/mlflow-apps", "mlflow", "mlflow-apps"]],
     ["http://github.com/mlflow/mlflow-apps.git", ["/github.com/mlflow/mlflow-apps", "mlflow", "mlflow-apps"]],
     ["https://github.com/mlflow/mlflow-apps.git", ["/github.com/mlflow/mlflow-apps", "mlflow", "mlflow-apps"]],
-    ["git@github.com:mlflow/mlflow-apps.git", ["/github.com/mlflow/mlflow-apps", "mlflow", "mlflow-apps"]],
-    ["https://some-other-site.com?q=github.com/mlflow/mlflow-apps.git", null],
-    ["ssh@some-server:mlflow/mlflow-apps.git", null],
-  ]
-  urlAndExpected.forEach(lst => {
+    ["git@github.com:mlflow/mlflow-apps.git", ["@github.com:mlflow/mlflow-apps", "mlflow", "mlflow-apps"]],
+    ["https://some-other-site.com?q=github.com/mlflow/mlflow-apps.git", [null]],
+    ["ssh@some-server:mlflow/mlflow-apps.git", [null]],
+  ];
+  urlAndExpected.forEach((lst) => {
     const url = lst[0];
     const match = url.match(gitHubRegex);
-    expect(match == lst[1]);
-  })
-})
+    expect([].concat(match)).toEqual(lst[1]);
```

Comment:
This test was actually broken before: `expect(a == b)` doesn't work, since `expect` just wraps an object so you can call things like `toEqual`. I had to change it to use `[].concat` here to turn the match object into an array, and I also had to change some of the expected values above.